redisinsight/ui/src/components/query-card/QueryCardHeader/QueryCardHeader.tsx

```diff
@@ -235,6 +237,30 @@ const QueryCardHeader = (props: Props) => {
     }
   })
 
+  const profileOptions: EuiSuperSelectOption<any>[] = (getProfileViewTypeOptions() as any[]).map((item) => {
+    const { value, id, text } = item
+    return {
+      value: id ?? value,
+      inputDisplay: (
+        <div className={cx(styles.dropdownOption, styles.dropdownProfileOption)}>
+          <EuiIcon
+            className={styles.iconDropdownOption}
+            type={'visTagCloud'}
+            data-testid={`view-type-selected-${value}-${id}`}
+          />
+        </div>
+      ),
+      dropdownDisplay: (
+        <div className={cx(styles.dropdownOption, styles.dropdownProfileOption)}>
+          <span>{truncateText(text, 20)}</span>
+        </div>
+      ),
+      'data-test-subj': `profile-type-option-${value}-${id}`,
+    }
+  })
+
+  const canCommandProfile = isCommandAllowedForProfile(query.split(' ')[0].toLowerCase())
```

Comment:
I think it's better to move `(' ')[0].toLowerCase()` to the `isCommandAllowedForProfile` to not duplicate this everywhere in future
and it will be enough just use `isCommandAllowedForProfile(query)` in the condition below